### PR TITLE
My-Portfolio

### PR DESCRIPTION
**Pull request by mesfin345**
I create a portfolio page :
     -a mobile version Skelton portfolio page

### CODE REVIEW

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @mesfin345 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that still needs the following requirement:
- [ ] Kindly set up your linters correctly by following this [linter template](https://github.com/microverseinc/linters-config/tree/master/html-css)
- [ ] Please create a descriptive PR title and a short summary describing changes.
- [ ] Please add a descriptive readme file by using the microverse [readme template](https://github.com/microverseinc/readme-template).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by mesfin345** [comment]
I set up linters correctly
and add a descriptive PR and readme file

**Comment by Amrhub** [comment]
Hi  @mesfin345, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, the Readme file should contain all main headers that fit your project from [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) I know it was not yet so clear! lets make sure we fix that to be able to continue on with the review 👍🏻 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # Portfolio
+
 A clean and clear portfolio page
+
+# To make the portfolio page
+
+    *using -Html and css
+          -using Vs Code a text editor
```
Comment (CollinsTatang) [inline README.md:8]: - [ ] Please make your readme file more descriptive by following the given template above

File: index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" class="header-logo">Mesfin</a>
+      <nav class="header-menu-btn mobile">
+        <div class="menu-btn__ham one"></div>
+        <div class="menu-btn__ham two"></div>
+        <div class="menu-btn__ham three"></div>
+      </nav>
+    </header>
```
Comment (CollinsTatang) [inline index.html:18]: Nice job, good use of sematics

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @mesfin345 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Gitflow flow was correctly used ✔️ 
- Design similar to Figma design template ✔️

## Required Changes ♻️

image

- [ ] Please give a meaningful name to your feature branch.
- [ ] Kindly give a descriptive PR title and a good summary of all changes.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mesfin345** [comment]
This is a project of my responsive portfolio website that e that will hold my professional information and various projects

File: index.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" class="header-logo">Mesfin</a>
+      <nav class="header-menu-btn mobile">
+        <div class="menu-btn__ham one"></div>
+        <div class="menu-btn__ham two"></div>
+        <div class="menu-btn__ham three"></div>
```
Comment (AnselemOdims) [inline index.html:16]: - [ ] [OPTIONAL] You should strongly consider using li tags for these and wrapping them in a ul tag

**Review by AnselemOdims — APPROVED** [review]
### Hi  @mesfin345 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions
- [ ] [OPTIONAL] You should consider making your pull request title and summary more descriptive
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**